benchmarking/factual_consistency_de.ipynb

```diff
@@ -0,0 +1,235 @@
+    "    score and the human annotation scores.'''\n",
+    "    # Ignore any data points where the evaluator returned `None`. This may happen\n",
+    "    # if, for example, the prompt triggers Azure OpenAI's content filter.\n",
+    "    result_df = result.to_df()\n",
+    "    indices = list(result_df[result_df['metric_value'].notna()].index)\n",
+    "    valid_metric_values = [result.metric_values[i] for i in indices]\n",
+    "    valid_annotated_scores = [annotated_scores[i] for i in indices]\n",
+    "\n",
+    "    pearson_corr = pearsonr(valid_metric_values, valid_annotated_scores)[0]\n",
+    "    spearman_corr = spearmanr(valid_metric_values, valid_annotated_scores)[0]\n",
+    "    kendalltau_corr = kendalltau(valid_metric_values, valid_annotated_scores)[0]\n",
+    "\n",
+    "    print(f'Pearson correlation = {pearson_corr}')\n",
+    "    print(f'Spearman correlation = {spearman_corr}')\n",
+    "    print(f'Kendall-Tau correlation = {kendalltau_corr}')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Compute the factual consistency scores on QAGS-XSUM using the local (UniEval)\n",
+    "# model option and measure various correlations with the human annotated scores\n",
+    "from langcheck.metrics.de import factual_consistency\n",
+    "\n",
+    "result = factual_consistency(qags_xsum_generated_outputs, qags_xsum_sources)\n",
+    "compute_correlation_values(result, qags_xsum_scores)\n",
+    "\n",
+    "# RUN-DATE: 2024-1-17\n",
+    "# Resulting correlation values:\n",
+    "# Pearson correlation = 0.40358016311552586\n",
+    "# Spearman correlation = 0.37558373934197853\n",
+    "# Kendall-Tau correlation = 0.3097142857142857\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Compute the factual consistency scores on QAGS-XSUM using the OpenAI\n",
+    "# (gpt-3.5-turbo) model option and measure various correlations with the human\n",
+    "# annotated scores\n",
+    "from langcheck.metrics.de import factual_consistency\n",
+    "import os\n",
+    "\n",
+    "result = factual_consistency(qags_xsum_generated_outputs,\n",
+    "                             qags_xsum_sources,\n",
+    "                             model_type='azure_openai',\n",
+    "                             openai_args={'model': 'YOUR_DEPLOYMENT_NAME'})\n",
+    "\n",
+    "compute_correlation_values(result, qags_xsum_scores[:50])\n",
+    "\n",
+    "# RUN-DATE: 2024-1-17\n",
+    "# OpenAI deployment details:\n",
+    "# - Model name: gpt-35-turbo\n",
+    "# - Model version: 0613\n",
+    "# Resulting correlation values:\n",
+    "#   Computed on 230 examples\n",
+    "# Pearson correlation = 0.1632062194597614\n",
+    "# Spearman correlation = 0.15952417117218096\n",
+    "# Kendall-Tau correlation = 0.15103303151237832\n"
```

Comment:
Wow these are super low 😮 Do you have a sense of why it's so much worse than the English factual consistency?

Reply:
I know, that's why I had added the other cells while trying to find out why...
I am not sure if I did something wrong. When I spot checked the results:
a) the translation did not seem to be an issue, and the explanation made sense
and
b) the results actually matched my own impression of the results...

I would re-run this with the translated prompts

Reply:
Re-running it post prompts update is much better:
```
# Pearson correlation = 0.4101880067233937
# Spearman correlation = 0.4099039059599361
# Kendall-Tau correlation = 0.3868346648563296
```